ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -51022,6 +51022,13 @@ periodics:
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-ovn-upgrade-rollback-oldest-supported
+  reporter_config:
+    slack:
+      channel: '#alerts-ocp-rollback-ci'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: Job {{.Spec.Job}} failed, see {{.Status.URL}}
```

Comment:
If we are reporting both `failure` and `error`, it might be worth including `{{.Status.State}}` here, like:

    Job {{.Spec.Job}} ended with {{.Status.State}}, see <{{.Status.URL}}|here>.
 
 or some such.  `<{{.Status.URL}}|View logs>` seems more popular grepping for existing `report_template` content, and personally there is more in the job-details page in logs, so I like `here`.  But it's not pinging me, so :shrug:, feel free to take this feedback and do whatever you want with it, including leaving the string you're currently using in place.
 
 /approve
 /hold in case you did want to make changes